shared/Utils.js

```diff
@@ -3,5 +3,57 @@
 export default {
 	stripHtmlTags (inputString) {
 		return inputString.replace(/<\/?[^>]+(>|$)/g, "");
+	},
+
+	/**
+	 * The complete Triforce, or one or more components of the Triforce.
+	 * @typedef {Object} menuItem
```

Comment:
I think these are supposed to be HeadsUp camelcase. Although I'm not 100% sure of that.